src/qt/emuwindow.cpp

```diff
@@ -1,7 +1,9 @@
 #include <fstream>
 #include <iostream>
+#include <tgmath.h>
```

Comment:
Two things:
* Use ```ctgmath``` instead of ```tgmath.h```. I'll need to add this to the style guide.
* C++ specifications say ```tgmath.h``` includes both ```math.h``` and ```complex.h``` (in this case, ```cmath``` and ```ccomplex```). Why not just use ```cmath```?